packages/backend/helpers/redis.ts

```diff
@@ -0,0 +1,8 @@
+import { createClient } from 'redis';
+import config from '../config';
+
```

Comment:
@Hvinder lets move this out to its own folder? Perhaps `backend/redis/client/index.ts`